lib/bot.js

```diff
@@ -200,33 +220,50 @@ class Bot {
       if (!discordChannel) {
         logger.info('Tried to send a message to a channel the bot isn\'t in: ',
           discordChannelName);
-        return;
+        return null;
       }
 
-      const withMentions = text.replace(/@[^\s]+\b/g, (match) => {
-        const search = match.substring(1);
-        const guild = discordChannel.guild;
-        const nickUser = guild.members.find('nickname', search);
-        if (nickUser) {
-          return nickUser;
-        }
+      return discordChannel;
+    }
+    return null;
+  }
 
-        const user = this.discord.users.find('username', search);
-        if (user) {
-          const nickname = guild.members.get(user.id).nickname;
-          if (!nickname || nickname === search) {
-            return user;
-          }
+  sendToDiscord(author, channel, text) {
+    const discordChannel = this.getDiscordChannelFor(channel);
+    if (!discordChannel) return;
+
+    const withMentions = text.replace(/@[^\s]+\b/g, (match) => {
+      const search = match.substring(1);
+      const guild = discordChannel.guild;
+      const nickUser = guild.members.find('nickname', search);
+      if (nickUser) {
+        return nickUser;
+      }
+
+      const user = this.discord.users.find('username', search);
+      if (user) {
+        const nickname = guild.members.get(user.id).nickname;
+        if (!nickname || nickname === search) {
+          return user;
         }
+      }
 
-        return match;
-      });
+      return match;
+    });
 
-      // Add bold formatting:
-      const withAuthor = `**<${author}>** ${withMentions}`;
-      logger.debug('Sending message to Discord', withAuthor, channel, '->', discordChannelName);
-      discordChannel.sendMessage(withAuthor);
-    }
+    // Add bold formatting:
+    const withAuthor = `**<${author}>** ${withMentions}`;
+    logger.debug('Sending message to Discord', withAuthor, channel, '->', `#${discordChannel.name}`);
+    discordChannel.sendMessage(withAuthor);
+  }
+
+  /* Sends a message to Discord exactly as it appears */
+  sendSpecialToDiscord(channel, text) {
```

Comment:
Never mind, I was a bit confused since I forgot that we append usernames to the messages even when sent to discord (slack-irc doesn't). The current approach makes sense. Could maybe call it something like `sendExactToDiscord` though. Also, `getDiscordChannelFor` -> `findDiscordChannel`/`getDiscordChannel` or something like that?